html/js/common.js

```diff
@@ -5,4 +5,20 @@ function redirect(url, http){
   else{
     window.location.href='https://'+url;
   }
-}
\ No newline at end of file
+}
+
+function sendUserTime(){
+  var userDate = new Date();
+  var moth = userDate.getUTCMonth();
+  var userTime = userDate.getFullYear() + "-" + String(userDate.getMonth()+1) + "-" + userDate.getDate() + " " + userDate.getHours() + ":" + userDate.getMinutes() + ":" + userDate.getSeconds();
```

Comment:
@iwanicki92 Or maybe we shall set timezone as well?

Reply:
`var moth` was used for debug, good point, deleted.

Instead of setting UTC time (which is not on API of timedatectl?) first set timezone, then set local time, should work the same.

Reply:
If Wiktor adds timezones then ok, if not then just get UTC time.

Reply:
Auto timezone set added.